.github/copilot-instructions.md

```
@@ -1,18 +1,40 @@
 # VinaAcademy Frontend - AI Coding Instructions
 
+<div style="background: linear-gradient(135deg, rgba(84, 180, 211, 1) 0%, rgba(57, 192, 237, 0.2) 100%); padding: 16px; border-radius: 8px; margin-bottom: 24px;">
+  <strong>🎯 Quick Navigation:</strong> Next.js 15+ • App Router • Microservices • Real-time Chat & Notifications • Role-based Access
+</div>
+
 ## General Guidelines
 
-**IMPORTANT**: Do NOT create markdown documentation files (e.g., README.md, SETUP.md, GUIDE.md, CHANGES.md) after completing tasks unless explicitly requested by the user. Focus on implementing the actual code changes requested. Provide a brief summary in the chat instead.
+> [!IMPORTANT]
+> **CRITICAL**: Do NOT create markdown documentation files (e.g., README.md, SETUP.md, GUIDE.md, CHANGES.md) after completing tasks unless explicitly requested by the user. Focus on implementing the actual code changes requested. Provide a brief summary in the chat instead.
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)